src/pages/auth/LoginForm.tsx

```diff
@@ -67,8 +67,9 @@ export const LoginForm = ({
       />
 
       <Spacer size={32} />
-
-      <Button type="submit" loading={isLoading}>
+      <Text>Remember me</Text>
+      <Spacer size={32} />
+      <Button type="submit" loading={isLoading} css={{width: '100%', backgroundColor: '$system-black', color: '$system-white'}}>
```

Comment:
1. Leave this button yellow or add another variant (black button) into <Button> component
2. There is full prop to make it full width

Reply:
Fixed